habitat-baselines/habitat_baselines/rl/ver/environment_worker.py

```diff
@@ -339,7 +337,8 @@ def _create_worker_configs(config: "DictConfig"):
 
     # We use a minimum number of scenes per environment to reduce bias
     scenes_per_env = max(
-        int(math.ceil(len(scenes) / num_environments)), MIN_SCENES_PER_ENV
+        int(math.ceil(len(scenes) / num_environments)),
+        config.habitat_baselines.min_scenes_per_env,
```

Comment:
```suggestion
        config.habitat_baselines.rl.ver.min_scenes_per_env,
```

Reply:
Just tested, `config.habitat_baselines.min_scenes_per_env,` does not work with habitat-lab `v0.2.4` but  `config.habitat_baselines.rl.ver.min_scenes_per_env` works fine.